core/src/main/java/hudson/console/AnnotatedLargeText.java

```diff
@@ -113,7 +113,7 @@ protected void setContentType(StaplerResponse rsp) {
         rsp.setContentType(isHtml() ? "text/html;charset=UTF-8" : "text/plain;charset=UTF-8");
     }
 
-    private ConsoleAnnotator createAnnotator(StaplerRequest req) throws IOException {
+    private ConsoleAnnotator<T> createAnnotator(StaplerRequest req) throws IOException {
```

Comment:
Most of the patch is correcting or suppressing rawtypes/unchecked warnings. Will call out the two actual fixes explicitly.